front/src/common-app/index.ts

```diff
@@ -1 +1,7 @@
-export * from './base-action';
\ No newline at end of file
+export * from "./base-action";
+
+export const actionIds = {
+  START_READ_COLLECTION_USERS: "[0] Start read collection users.",
+  COMPLETED_READ_COLLECTION_USERS: "[1] Completed read collection users.",
+  INSERT_NEW_USER: "[2] Insert a new user completed."
```

Comment:
Should it be INSERT_NEW_USER_REQUEST_START ? May we will add in the future a completed one to handle the result and errors